### PR TITLE
feat: add cockpit legacy interaction command

### PR DESCRIPTION
**Pull request by oim1**

### CODE REVIEW

**Review by Valastiri — CHANGES_REQUESTED** [review]
-- Not a code review --

Please re-title the embed more along the lines of  `MSFS | Cockpit Interaction System` as Legacy control is a bit generic and may get confused with other legacy items. 

Additional Reference -> https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/#cockpit-interaction-system 

May be prudent to give users options as the aircraft is serviceable using the other method as well.

**Review by Valastiri — APPROVED** [review]

**Comment by Valastiri** [comment]
-- Approved the copy -- 

Someone else for code.

File: src/commands/controls.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'MSFS | Cockpit Interaction System',
+        fields: [
+            {
+                name: 'Legacy Cockpit Interaction System',
+                value: '> Go to menu'
+                        + '\n> General Options'
+                        + '\n> Accessibility'
+                        + '\n> Find the Cockpit Interaction System setting'
+                        + '\n> Change to legacy',
```
Comment (Benjozork) [inline src/commands/controls.ts:18]: Use `makeLines` function like in other commands

File: src/commands/controls.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            {
+                name: 'Using the new cockpit interaction system',
+                value: '> Highlight a control (like a knob)'
+                        + '\n> Hold left click to lock that control. Now your mouse will not affect other controls or other mouse bindings'
+                        + '\n> Move the mouse left to turn the knob left, move it right to turn the knob to the right (with left click held down)'
+                        + '\n> You can also use the scroll wheel while holding left click down to turn the knob to the left or right'
+                        + '\n> To push a control/knob in, lock to the control using left click and then right click'
+                        + '\n> To pull a control/knob out, hold left click and then click your scroll wheel (middle mouse)'
+                        + '\n> Note: If you already use middle mouse button to activate freelook this may not work.'
+                        + ' Check your keybinds so this feature does not conflict',
```
Comment (Benjozork) [inline src/commands/controls.ts:30]: Use `makeLines` function like in other commands

File: src/commands/index.ts [hunk 3]
```diff
@@ -25,6 +25,7 @@ import { utf8 } from './utf-8';
 import { calibrate } from './calibrate';
 import { CommandDefinition } from '../lib/command';
 import Logger from '../lib/logger';
+import { controls } from './controls';
```
Comment (Benjozork) [inline src/commands/index.ts:28]:
```suggestion
import { controls } from './controls';
import Logger from '../lib/logger';
```

**Review by Benjozork — CHANGES_REQUESTED** [review]

File: src/commands/index.ts (hunk 3 above)
Comment (benw202) [reply src/commands/index.ts:28]: Can we move import { controls } etc. above import { CommandDefinition } etc. too?

It's nice to have the commands above other imports for clarity. Like in previous PR's

**Comment by Benjozork** [comment]
Have you tested this yourself ?

**Comment by oim1** [comment]
Yes, I have tested this.

**Review by Benjozork — APPROVED** [review]

**Pull request merged**